doc/webapp.md

```diff
@@ -0,0 +1,94 @@
+- webapp
+    - Main.js
+    - Routes.js
+    - components
+    - main
+        - style
+        - middleware
+        - store.js
+    - service
+        - api
+        - socket
+    - utils
+    - views
+        - guest
+        - app
+            - index.js
+            - AppView.js
+            - AppView.scss
+            - components
+                - header
+                - job
+                - sideBar
+            - views
+                - home
+                    - index.js
+                    - HomeView.js
+                    - HomeView.scss
+                    - views
+                        - collectImportReport
+                        - dashboard
+                        - surveyCreate
+                        - surveyList
+                - designer
+                - data
+                - analysis
+                - user
+                - table
+                - survey
+                    - categories
+                    - category
+                    - categorySelector
+                    - cyclesSelect
+                    - items
+                    - labelsEditor
+                    - nodeDef
+                    - nodeDefsSelector
+                    - record
+                    - surveyDefsLoader
+                    - surveyForm
+                    - taxonomies
+                    - taxonomy
+    - store
+        - app
+            - dialogConfirm
+            - errors
+            - job
+            - notification
+            - i18n
```

Comment:
I am not sure it is the moment and the right place,

 I saw in the repo that all of the translations are into the same file, how about split it based on the view or context?

- i18n
- -  en
- - - Survey
- - - Designer
- - it
- - - ...

Reply:
yes I know, it's already in the refactor work plan. here only webapp code refactor